### PR TITLE
[TTAHUB-2754] Add overview statistics widget for training report dashboard

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

This is a backend only ticket for the TR dashboard overview widget.

## How to test

- Confirm that the query evaluates only completed training reports, and that all other calculations are correct
- Ensure the test adequately tests the feature

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2754

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by AdamAdHocTeam — COMMENTED** [review]
I think you have the wrong ticket linked here 2574.

https://jira.acf.gov/browse/TTAHUB-2574

**Comment by thewatermethod** [comment]
> I think you have the wrong ticket linked here

ok thanks, fixed

File: src/widgets/trOverview.ts [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  }) as ITrainingReportForOverview[];
+
+  const data = reports.reduce((acc: IReportData, report) => {
+    const { sessionReports } = report;
+
+    let sessionGrants = [];
+    let sessionDuration = 0;
+    let sessionParticipants = 0;
+
+    sessionReports.forEach((sessionReport) => {
```
Comment (AdamAdHocTeam) [inline src/widgets/trOverview.ts:110]: Probably fine as is, but might be faster in sql with group by.

File: src/widgets/trOverview.test.js [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+    // - session report 1
+    await createSessionReport({
+      eventId: trainingReport1.id,
+      data: {
+        deliveryMethod: 'in-person',
+        duration: 1,
+        recipients: [{ value: grant1.id }, { value: grant2.id }],
+        numberOfParticipantsVirtually: 0,
+        numberOfParticipantsInPerson: 0,
```
Comment (AdamAdHocTeam) [inline src/widgets/trOverview.test.js:87]: Do we count 'numberOfParticipiantsInPerson'? If so not sure if you want to include it in the test data.

**Pull request merged**